Makefile

```diff
@@ -25,3 +25,8 @@ protogen: check-protoc check-gogofaster
 		./_tools/protogen_golang.sh
 		pip3 install $(PIP_ARGS) grpcio_tools==1.13.0
 		./_tools/protogen_python.sh
+
+test-python-sdk:
```

Comment:
let's add it to main `test` target so ci would actually run it :)
If I remember right `test: test-python-sdk` should do the trick.

Reply:
yup it did! thanks! 👍

Reply:
there's one thing that I don't understand thoug in .travis.yml. It seems that for tests requiring both python and go this is what should be done:
```
      name: 'Golang 1.10: generated code'
      stage: test
      language: go
      go: '1.10'
      env:
        - PYENV_VERSION="3.6.7"
      script:
        - make protogen
        - make no-changes-in-commit
```

but how that `PYENV_VERSION` works? I don't see any doc for that in travis nor any installation of pyenv.

Reply:
if I remember right go image on travis contains pyenv with some pre-installed python versions. And pyenv uses [PYENV_VERSION](https://github.com/pyenv/pyenv#choosing-the-python-version) env variable to choose which version of python to run.

Reply:
Ok, thanks! I rebased due to conflicts with .travis.yml.